master/getting-started/kubernetes/installation/gce.md

```diff
@@ -2,49 +2,21 @@
 title: Deploying Calico and Kubernetes on GCE
 ---
 
-These instructions allow you to set up a Kubernetes cluster with Calico networking on GCE using the [Calico CNI plugin][cni-plugin]. This guide does not setup TLS between Kubernetes components or on the Kubernetes API.
+There are several possibilities for [deploying Kubernetes on GCE][running-GCE-on-K8s]
```

Comment:
It was purposefully vague, we don't really care how Kubernetes gets installed.  If you feel like this is better it works for me.